module-2/backend/week_two.md

```diff
@@ -5,17 +5,77 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. At a high level, what is ActiveRecord? What does it do/allow you to do?
+
+  Active Record is an object relational mapper. It maps Ruby objects to a SQL database. ActiveRecord also provides a set of methods (called on a Ruby class) to query a SQL database (translates the ruby method to SQL).
+  
 2. What kind of methods are `belongs_to`, and `has_many`? (i.e. class or instance) Give an example.
+
+  They are class methods.
+  ```ruby
+  class Student
+    has_many :classes
+    belongs_to :school
+   end
+  ```
```

Comment:
Think this is a stale version, but right idea. If you're working on your own repo, be sure to set up this repo as an `upstream` repository and then pull the latest version before working on this.

`belongs_to` and `has_many` are class methods, but quick note: the methods they give you access to are instance methods. So, `.classes`, and `.school` in the example below have to be called on a specific instance of a student:

```
student1 = Student.find(1)
student1.classes
student1.school
```